docs/update_tails_usbs.rst

```diff
@@ -0,0 +1,104 @@
+Updating Tails USBs
+===================
+
+You are responsible for keeping your Tails USBs updated.
+
+There are two ways to perform updates: via the Tails graphical installer and manually. The manual update process will work on any Tails USB. The graphical installer requires an Internet connection to notify you when updates are available, so it is only suitable for internet-connected Tails workstations (such as the *Admin Workstation* and the *Journalist Workstation*).   
+
+Because the *Secure Viewing Station* (SVS) is airgapped, it cannot receive update notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the *Submission Private Key*), so we **strongly** encourage you to manually update the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 to 8 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 to 8 weeks: https://tails.boum.org/contribute/calendar/
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Update via Graphical Installer
+------------------------------
+
+For Internet-connected Tails workstations, you'll see a notification when a new version of Tails is available. 
+
+|Update Notification| 
+
+We recommend that you back up your existing configuration and `update as soon as possible`_. It usually takes some time for updates to download, so keep that in mind when choosing when to update.
```

Comment:
Link should probably be on "back up your existing configuration", not "update as soon as possible. Also, the link should be internal rather than linking to the stable version of the docs.

Reply:
fixed in 6582bc5f4